fix(doc): fix out-of-date info in active-directory-or-ldap-authentication

jjargot opened the pull request:
standalone/configuration/standalone.xml => /setup/wildfly-templates/standalone.xml
/bin/setenv.sh => /setup/tomcat-templates/setenv.sh
/bin/setenv.bat => /setup/tomcat-templates/setenv.bat

Versions: 7.4, 7.5, 7.6

jjargot committed:
fix(doc): fix out-of-date info

standalone/configuration/standalone.xml => /setup/wildfly-templates/standalone.xml
/bin/setenv.sh => /setup/tomcat-templates/setenv.sh
/bin/setenv.bat => /setup/tomcat-templates/setenv.bat

Versions: 7.4, 7.5, 7.6

ugaston reviewed (commented):
Nice.
But I believe you should also replace:
- <TOMCAT_HOME>/conf --> <TOMCAT_HOME>/server/conf
- <TOMCAT_HOME>/bin --> <TOMCAT_HOME>/server/bin

jjargot committed:
fix(doc): implement Unai remarks


    <TOMCAT_HOME>/conf --> <TOMCAT_HOME>/server/conf
    <TOMCAT_HOME>/bin --> <TOMCAT_HOME>/server/bin

ugaston reviewed (approved):
Nice, thanks!
